Demo/Configuration.swift

```diff
@@ -83,9 +83,17 @@ internal struct Configuration: Codable {
     )
     
     fileprivate static func loadConfiguration() -> Configuration {
-        UserDefaults.standard.data(forKey: defaultsKey)
-            .flatMap { try? JSONDecoder().decode(Configuration.self, from: $0) }
-            ?? defaultConfiguration
+        if CommandLine.arguments.contains("SY") {
+           return Configuration(countryCode: "SY",
```

Comment:
We can fix it another PR:

Maybe we can use `countryCode` and `currencyCode` as variables ?
If tomorrow we decide to rewrite tests for Blik(LPM from Poland) - do we need to add 
`if CommandLine.arguments.contains("PO") { return Configuration(...) }` ?